ceph/ceph_cluster.yml

```diff
@@ -6,25 +6,29 @@ parameters:
  prefix: ceph
  numcpus: 2
  memory: 4096
```

Comment:
I'd also try with less memory (2GB or even 1GB). 4 GB x 3 nodes means 12 GB just for this, which makes it hard to fit into a 16 GB machine.

Reply:
Reduced to 1 CPU and 2GB